src/rpc/blockchain.cpp

```diff
@@ -1070,9 +1073,10 @@ UniValue gettxout(const JSONRPCRequest& request)
     CCoinsViewCache* coins_view = &::ChainstateActive().CoinsTip();
 
     if (fMempool) {
-        LOCK(mempool.cs);
-        CCoinsViewMemPool view(coins_view, mempool);
-        if (!view.GetCoin(out, coin) || mempool.isSpent(out)) {
+        const auto& tx_pool = EnsureMemPool();
```

Comment:
I think in a future where running without a mempool is possible, we could make this friendlier by just defaulting `include_mempool` to false for nodes without mempools rather than forcing the user to always pass false as an argument.

(no change needed for this PR but something to add when `-nomempool` is added as an option)